<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ariana_seat</class>
 <widget class="QDialog" name="ariana_seat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #e6e6e6;
color:black;</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>721</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/img/img/ariana_grande.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold;
font-size:24px;
color:#333333</string>
    </property>
    <property name="text">
     <string>Venue :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>133</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold;
font-size:24px;
color:#333333</string>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold;
font-size:24px;
color:#333333</string>
    </property>
    <property name="text">
     <string>Zone:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold;
font-size:24px;
color:#333333</string>
    </property>
    <property name="text">
     <string>Number of seat(s):</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_zone">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
font:bold;
background-color: white;
border-style: outset;
border-width: 2px;
border-color: black;
</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_seat">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>240</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
font:bold;
background-color: white;
border-style: outset;
border-width: 2px;
border-color: black;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>300</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton{
color:white;
font:bold;
font-size:20px;
background-color:red;
border-radius:5px;}

#pushButton:hover{
background-color:#b30000}</string>
    </property>
    <property name="text">
     <string>Make Payment</string>
    </property>
   </widget>
   <widget class="QLabel" name="number_seat">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green;
font:bold</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>451</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#333333;
border-radius:5px
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>451</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#333333;
border-radius:5px
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold;
font-size:24px;
color:#333333</string>
    </property>
    <property name="text">
     <string>Ariana Grande : Thank,u Next tour</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_venue">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>96</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font-size:18px;
color:#333333</string>
    </property>
    <property name="text">
     <string>Impact Exhibition Hall 5,Muangthong Thani</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_date">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>128</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;
color:#333333</string>
    </property>
    <property name="text">
     <string>7th November 2019 19:30</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>315</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_2{
color:#999999;
font:bold;
font-size:20px;
background-color:white;
border-radius:5px;}

#pushButton_2:hover{
color:#666666}</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#b30000;
font:bold;
font-size:28px;</string>
   </property>
   <property name="text">
    <string>Select Seats</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
